Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
This have been discussed with @ilovezfs and we agreed that we need some warnings when we try to pour bottle with non-stable dependencies, since the bottle is actually built against stables.